File: URI/c++/Triângulo - 1043.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main(){
	float A, B, C;
	cin >> A >> B >> C;
	float P = A + B + C;
	cout << setprecision(1) << fixed;
	if(abs(B - C) <= A && A <= (B + C) && abs(A - C) <= B && B <= (A + C) && abs(A - B) <= C && C <= (A + B)){
		P /= 2;
		float area = sqrt(P * (P - A) * (P - B) * (P - C));
		cout << "Area = " << area << endl;
	} else {
		cout << "Perimetro = " << P << endl;
	}
	return 0;
}

File: URI/python/Triângulo - 1043.py
import math

A, B, C = (round(float(x), 1) for x in input().split())
if abs(B - C) < A < B + C or abs(A - C) < B < A + C or abs(A - B) < C < A + B:
    P = (A + B + C) / 2
    area = math.sqrt(P * (P - A) * (P - B) * (P - C))
    print(area)
else:
    print(A + B + C)